integration_tests/pages/page.ts

```diff
@@ -79,9 +79,7 @@ export default abstract class Page {
   }
 
   shouldContainButtonLink(text: string, href: string): void {
-    cy.get('.govuk-button').then(buttonElement => {
-      const { actual, expected } = helpers.parseHtml(buttonElement, text)
-      expect(actual).to.equal(expected)
+    this.shouldContainButton(text).then(buttonElement => {
       cy.wrap(buttonElement).should('have.attr', 'href', href)
     })
```

Comment:
Nice. Is it worth moving this commit before the last? I was a bit unsure what the use case was for the new method until reading this commit message